components/brave_sync/version_helper.cc

```diff
@@ -0,0 +1,68 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_sync/version_helper.h"
+
+#include <map>
+#include <string>
+#include <vector>
+
+#include "base/strings/string_split.h"
+#include "base/version.h"
+
+namespace {
+
+using MajorToFull = std::map<int, base::Version>;
+const MajorToFull can_self_delete_from{
+    {18, base::Version({1, 18, 2})},  // Nightly which actually can self delete
+    // Below are for test, fill with actual when uplifted
+    // {16, base::Version({1, 16, 100})},  // Beta & Dev
+    // {15, base::Version({1, 15, 100})},  // Release
+
+    // For the future
+    {19, base::Version({1, 19, 0})},
```

Comment:
Indeed, you are right.

Reply:
not applicable, because now the flag is used `is_self_delete_supported`, https://github.com/brave/brave-core/pull/6732/commits/c991c510d24eac8fef005c7bb4143bb871ecab7a